assets/js/datasets.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
+
+window.linqs = window.linqs || {};
+window.linqs.pubs = window.linqs.pubs || {};
+window.linqs.datasets = window.linqs.datasets || {};
+window.linqs.datapubs = window.linqs.datapubs || {};
+
+// Modified from pubs.js
```

Comment:
In general, all comments should be complete sentences.

This particular comment is unnecessary.